server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheContainerConfigurationBuilder.java

```diff
@@ -229,11 +245,34 @@ public GlobalConfiguration getValue() {
         return builder.build();
     }
 
+    private ClassLoader makeAggregatedClassLoader(ModuleLoader moduleLoader, ModuleIdentifier cacheContainerModule, List<ModuleIdentifier> additionalModules) throws ModuleLoadException {
+        if (additionalModules != null) {
+            List<ClassLoader> classLoaders = new ArrayList<>();
```

Comment:
We could make this a `ClassLoader[]` that is initialized to `additionalModules.size` +1 if `cacheContainerModule` wasn't null.

Reply:
Both could be null. Creating a list that will contain ```module``` + ```modules``` will not look pretty, so I preferred to pass both ```module``` and ```modules``` as separate args to ```makeAggregatedClassLoader```.

Reply:
`additionalModules` in this section was already checked against null. But I am not picky about this, I am fine the way it is.

Reply:
I do not understand what you mean.